cms/conftest.py

```diff
@@ -24,6 +24,11 @@ def pytest_configure(config):
     """
     Do core setup operations from manage.py before collecting tests.
     """
+    if config.pluginmanager.hasplugin("pytest_jsonreport") or config.pluginmanager.hasplugin("json-report"):
+        config.pluginmanager.register(DeferPlugin())
+    else:
+        print("pytest did not register json_report correctly")
```

Comment:
Will this show up in the output of pytest runs? Should we log this instead?

Reply:
it shows up if -v flag is set, but I should and will change this to logging instead